mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeans.scala

```diff
@@ -156,20 +157,33 @@ class BisectingKMeans private (
   private[spark] def run(
```

Comment:
do we still need this method `def run(input: RDD[Vector], instr: Option[Instrumentation])` ?

Reply:
Yes. It is called by this 
```
  @Since("1.6.0")
  def run(input: RDD[Vector]): BisectingKMeansModel = {
    run(input, None)
  }
```